docs/src/app.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%sveltekit.assets%/favicon.png" />
+    <link rel="icon" type="image/x-icon" href="/favicon.png" />
```

Comment:
@gesquinca I think we may want to keep `%sveltekit.assets%` ?
```suggestion
    <link rel="icon" type="image/x-icon" href="%sveltekit.assets%/favicon.png" />
```

Reply:
From what I can see, it doesn't work correctly with `%sveltekit.assets%/`. That's why I removed it. To try and see if it works correctly without it.